src/scss/styles-0-master.scss

```diff
@@ -7,7 +7,7 @@ html {
 
 h1, h2, h3, h4, button, input, optgroup, select, textarea {
   font-family: sans-serif;
-  font-weight: 300;
+  font-weight: 400;
```

Comment:
did you intentionally change this?

Reply:
Yes, weights looked light for h3's in particular